webapp/src/components/buttons/iconButton/index.tsx

```diff
@@ -22,6 +24,10 @@ const IconButton = ({tooltipText, iconClassName, extraClass = '', iconColor, onC
                 variant='outline-danger'
                 className={`plugin-btn button-wrapper btn-icon ${extraClass}`}
                 onClick={onClick}
+                aria-label={tooltipText}
+                role='button'
+                tabIndex={0}
+                onKeyDown={() => onPressingEnterKey(event, () => onClick?.())}
```

Comment:
```suggestion
                onKeyDown={(event) => onPressingEnterKey(event, () => onClick?.())}
```

Reply:
added it, but even without this it was working I guess because it was picking by default the event object